root/static/scripts/edit/URLCleanup.js

```diff
@@ -451,6 +451,7 @@ type CleanupEntries = {
   +[type: string]: CleanupEntry,
 };
 
+/* eslint-disable sort-keys */
```

Comment:
We disable this rule so often that I wonder if there's even a point to have it enabled by default across the codebase.

Reply:
Well, I like that we have a consistent sorting, either because we sort for this, or because we unset it because we want a *different* consistent sorting :) But it's not the most important thing in the world.